votrfront/js/ZapisnyListSelector.js

```diff
@@ -2,16 +2,28 @@
 import { CacheRequester, Loading } from './ajax';
 import { Link, QueryContext, queryConsumer } from './router';
 import { sortAs } from './sorting';
+import { currentAcademicYear } from './coursesStats';
 
+var buttonNovyZapisnyList = false;
```

Comment:
Globalna premenna?

Reply:
Ano, toto bolo nieco ako globalna premenna. Ako to chceme? Chceme vratit dvojicu alebo chceme na to zvlast funkciu?